packages/firestore/src/local/local_documents_view.ts

```diff
@@ -288,9 +284,16 @@ export class LocalDocumentsView {
         }
       }
     }
-    return this.remoteDocumentCache.getEntries(
-      transaction,
-      missingBaseDocEntriesForPatching
-    );
+    let mergedDocuments = existingDocuments;
```

Comment:
Question: the only use of `existingDocuments` is to initialize `mergeDocuments`. Is the idea to avoid modifying a function parameter?

Reply:
No, it's mainly there to be more readable: explicit about the merging happening.

I don't feel very strong about this though, i can just use `existingDocuments` if you think it's better.

Reply:
I don't feel strongly either way. If you've already started with the iOS port, just use whatever is more similar to the iOS code (in general, we try to keep the code as similar as practical across platforms).